src/3d/qgscolorramptexture.h

```diff
@@ -0,0 +1,76 @@
+/***************************************************************************
+                         qgscolorramptexture.h
+                         -------------------------
+    begin                : january 2020
+    copyright            : (C) 2020 by Vincent Cloarec
+    email                : vcloarec at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSCOLORRAMPTEXTURE_H
+#define QGSCOLORRAMPTEXTURE_H
+
+/// @cond PRIVATE
+
+#include <Qt3DRender/QEffect>
+#include <Qt3DRender/QGraphicsApiFilter>
+#include <Qt3DRender/QParameter>
+#include <Qt3DRender/QTexture>
+
+#include <QUrl>
+#include <QVector2D>
+#include <QVector3D>
+#include <QVector4D>
+#include <Qt3DRender/QBuffer>
+#include <QByteArray>
+
+#include "qgsmeshlayer.h"
+#include "qgsmeshlayerutils.h"
+#include "qgstriangularmesh.h"
+#ifndef SIP_RUN
```

Comment:
This is a private implementation file - so let's use `#define SIP_NO_FILE` here - and remove python/3d/auto_generated/symbols/qgspointcloud3dsymbol.sip.in file from git